tests/utils.go

```diff
@@ -469,6 +469,64 @@ func GetSupportedCPUModels(nodes k8sv1.NodeList) []string {
 	return cpus
 }
 
+func GetSupportedCPUFeaturesAmongAllTheNodes(nodes k8sv1.NodeList) []string {
+	var featureDenyList = map[string]bool{
```

Comment:
IMO, the type can be improved; if you're really just implementing a set, you can make it a map of `map[string]struct{}` , which is preferable, since an empty struct occupies no memory.

Reply:
Oh i didn't know this trick nice!
Done.